src/main.rs

```diff
@@ -123,8 +123,12 @@ impl ClippyCmd {
             .map(|arg| format!("{}__CLIPPY_HACKERY__", arg))
             .collect();
 
+        // Currently, `CLIPPY_TERMINAL_WIDTH` is used only to format "unknown field" error messages.
+        let terminal_width = termize::dimensions_stdin().map_or(0, |(w, _)| w);
```

Comment:
Good to know that might also come in handy for a different project :upside_down_face:

Reply:
> As I was typing this, I realized that dimensions_stdin should be dimensions in the line you cited, hence the forced push. (dimensions_stdin was a relic from experimenting with this very issue.)

Thanks for the hint. GitHub has finally improved their force push information so that it's possible to see the specific changes of a force push. :partying_face: